spec/models/book_spec.rb

```diff
@@ -1,12 +1,15 @@
 require 'rails_helper'
 
 describe Book do
-  subject(:book) { create(:book) }
+  subject(:book) { build(:book) }
 
-  context 'Create a book with Faker' do
-    it '#year is_string?' do
-      # It's strange but the model tells year is a String
-      expect(book.year).to be_instance_of(String)
-    end
+  it { is_expected.to be_valid }
+
+  context 'validates not null fields' do
+    it { book.validates_presence_of :genre }
```

Comment:
We don't use a context to group validations specs, move this examples to the same level of the `be_valid` spec!
Also, if you are making an expectation on the subject, you can use `is_expected` instead of use `book`. For example, this
```ruby
it { book.validates_presence_of :author }
```
can be changed to
```ruby
it { is_expected.to validates_presence_of(:author) }
```